src/core/Akka.Streams/Dsl/Sink.cs

```diff
@@ -270,8 +270,7 @@ public static class Sink
         /// </summary>
         /// <typeparam name="TIn">TBD</typeparam>
         /// <returns>TBD</returns>
-        public static Sink<TIn, Task> Ignore<TIn>()
-            => new Sink<TIn, Task>(new SinkholeSink<TIn>(Shape<TIn>("BlackholeSink"), DefaultAttributes.IgnoreSink));
+        public static Sink<TIn, Task> Ignore<TIn>() => FromGraph(new IgnoreSink<TIn>());
```

Comment:
I think Done only exists here because they always need to return something, like in F#, so we skipped it because it doesn't contain any useful information and returning Task without T is common in C#.